examples/applications/Perera_Science_Granda_Nature_Suzuki/Granda_Perera_ml_example.ipynb

```diff
@@ -0,0 +1,2515 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Example application of ORD datasets for downstream machine learning modeling"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Reproducing neural network predictor of reaction yields from Granda et al. Nature 2018, 559, 377, DOI: 10.1038/s41586-018-0307-8.<br>\n",
+    "Data from Perera et al. Science 2018, 359, 429, DOI: 10.1126/science.aap9112.<br>\n",
+    "ORD dataset location: https://github.com/open-reaction-database/ord-data/tree/main/data/68<br>\n",
+    "ORD dataset ID: ord_dataset-68cb8b4b2b384e3d85b5b1efae58b203"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 19,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Import packages\n",
+    "import ord_schema\n",
+    "from ord_schema import message_helpers, validations\n",
+    "from ord_schema.proto import dataset_pb2\n",
+    "\n",
+    "import math\n",
+    "import pandas as pd\n",
+    "import numpy as np\n",
+    "import tensorflow as tf\n",
+    "import matplotlib.pyplot as plt\n",
+    "import os\n",
+    "import wget\n",
+    "\n",
+    "from rdkit import Chem\n",
+    "from rdkit.Chem import AllChem\n",
+    "from sklearn.model_selection import train_test_split\n",
+    "from sklearn.metrics import r2_score\n",
+    "from glob import glob"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Download data from ord-data pull request and load to Dataset message\n",
+    "url = 'https://github.com/open-reaction-database/ord-data/blob/main/data/68/ord_dataset-68cb8b4b2b384e3d85b5b1efae58b203.pb.gz?raw=true'\n",
+    "\n",
+    "# Check if .pb.gz has already been downloaded\n",
+    "suffixes = ('.pb.gz', '.pb', '.pbtxt.gz', '.pbtxt')\n",
+    "pbs = [pb for pb in glob(os.getcwd()+'/*') if pb.endswith(suffixes)]\n",
+    "if len(pbs) > 0:\n",
+    "    pb = pbs[0]\n",
+    "else:\n",
+    "    pb = wget.download(url)\n",
+    "\n",
+    "# Load dataset message\n",
+    "data = message_helpers.load_message(pb, dataset_pb2.Dataset)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "[]"
+      ]
+     },
+     "execution_count": 4,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Ensure dataset validates\n",
+    "valid_output = validations.validate_message(data)\n",
+    "\n",
+    "# Check errors\n",
```

Comment:
Since validate_message by default raises an exception on error, I think this cell could be simplified to just `validations.validate_message(data)`.

Reply:
Done, thanks!